src/hooks/use-import-export.tsx

```diff
@@ -101,9 +101,16 @@ export const ImportExportProvider = ( { children }: { children: React.ReactNode
 			try {
 				await stopServer( selectedSite.id );
 
+				let filePath: string;
+				if ( file instanceof File ) {
+					filePath = getIpcApi().showFilePath( file );
+				} else {
```

Comment:
Maybe we should move this logic to `src/hooks/use-add-site.ts` instead? Seems reasonable that this function would consume a single format, and `src/hooks/use-add-site.ts` already calls the IPC API in some other places, so it wouldn't be exotic to also introduce a `getIpcApi().showFilePath` call

Reply:
It's really reasonable idea, but it's used more than in `src/hooks/use-add-site.ts`. There are 6 calls of this function, and that's why I decided to keep it flexible with expecting both`File | { path: ... }`. Otherwise, it would be just copy-paste. WDYT about it?
<img width="422" alt="Screenshot 2025-01-13 at 22 04 47" src="https://github.com/user-attachments/assets/26e771ef-8944-4d66-a751-ffba645b8f28" />

Reply:
I think `src/hooks/use-add-site.ts` is the only place where we pass a `File` instance, though.